### ISSUE TITLE
swupd manifests inconsistent with filesystem state for pcre2-10.34-23

### ISSUE THREAD

**Issue by gtkramer** [clearlinux/distribution#1589]
After doing a clean install of os-core-dev (or any of the ~40 other bundles that this impacts, most of which are -dev bundles), and then running `swupd diagnose` to make sure the system is consistent with swupd's view of the manifest, the operation fails with the following message:
```
Checking for extraneous files
-> File that should be deleted: /usr/lib64/libpcre2-32.so.0
-> File that should be deleted: /usr/lib64/haswell/libpcre2-32.so.0
```

Something is either wrong in the manifests or in the package update.

@otaviobp @reaganlo @fenrus75

**Comment by phmccarty** [comment]
The most recent `pcre2` package update needed an update for the `extras` file. This will be fixed in `pcre2-10.34-24`.

**Comment by phmccarty** [comment]
Fixed in release `32020`.
